### PR TITLE
New cordova plugin conventions added. SSZipArchive updated.

### PR DESCRIPTION
**Pull request by mlegenhausen**
Since cordova 5.0.0 npm is used for plugin management. I added a `package.json` and renamed the plugin in `cordova-plugin-chromium-zip`.

SSZipArchive source files are updated, to keep track with recent changes.

### CODE REVIEW

**Comment by mlegenhausen** [comment]
Please don't merge the SSZipArchive changes. Seems not to work properly will investigate.

**Comment by mlegenhausen** [comment]
SSZipArchive 0.2.0 integrated 0.2.1 has some memory overflow issues. New progress handler added, cause the old one was removed in 0.2.0.

Does now work with the updated SSZipArchive version.

File: zip.js [hunk 1]
```diff
@@ -2,20 +2,19 @@ var exec = cordova.require('cordova/exec');
 
 function newProgressEvent(result) {
     var event = {
-            loaded: result.loaded,
-            total: result.total
+        fileIndex: result.fileIndex,
+        totalFiles: result.totalFiles,
+        total: 1, // For backwards compability
```
Comment (agrieve) [inline zip.js]: I suspect the changes to the JS will break the Android part of the plugin. Have you tested that? Probably would be better to set these all on the native side.

Comment (mlegenhausen) [inline zip.js]: You are right off cause, had focued on iOS, will fix this when I come to android testing.

**Comment by mlegenhausen** [comment]
Merged your changes and reverted the result to be compatible with android again.

**Comment by agrieve** [comment]
Thanks! Merged it in and published to npm as v3.0.0

**Pull request closed without merging**